src/utils/array/ChunkArray/index.js

```diff
@@ -1,33 +1,4 @@
-export const ChunkArray = (arr, size) => {
-    const myArray = [];
-    for (let i = 0; i < arr.length; i += size) {
-        myArray.push(arr.slice(i, i + size));
-    }
-    return myArray;
-};
-
-export const chunkArrayCustom = (perChunk = 2, inputArray = []) => {
-    const result = inputArray.reduce((resultArray, item, index) => {
-        const chunkIndex = index % perChunk
-
-        if (!resultArray[chunkIndex]) {
-            resultArray[chunkIndex] = []; // start a new chunk
-        }
-        resultArray[chunkIndex].push(item);
-        return resultArray;
-    }, []);
-    return result
-};
-
-export const insertNewTopicIntoTopics = (newTopic, topics, setListTopic, setHashtags) => {
-    if (!topics.includes(newTopic)) {
-        const newArr = [...topics, newTopic];
-        setListTopic(newArr);
-        setHashtags(newArr)
-    }
-}
-
 export const joinTopicIntoTopicList = (newTopic = '', topics = []) => {
-    if (!topics?.includes(newTopic)) return [...topics, newTopic]
-    return topics
-}
\ No newline at end of file
+  if (!topics?.includes(newTopic)) return [...topics, newTopic];
+  return topics;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [1-5]

The function `joinTopicIntoTopicList` seems to be fine. However, it's important to note that the function will return a new array every time it is called, even if `newTopic` is already in `topics`. This could potentially lead to unnecessary re-renders if you're using this function in a React component and the returned value is part of the component's state or props. If this is the case, consider returning the original `topics` array when `newTopic` is already included.

```diff
-  if (!topics?.includes(newTopic)) return [...topics, newTopic];
+  if (!topics?.includes(newTopic)) return topics.concat(newTopic);
   return topics;
```
In this way, a new array is only created when necessary, otherwise, the original array is returned.


<!-- This is an auto-generated comment by OSS CodeRabbit -->